server/build/webpack.js

```diff
@@ -75,8 +75,17 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
     )
   }
 
-  const babelRuntimePath = require.resolve('babel-runtime/package')
-  .replace(/[\\/]package\.json$/, '')
+  const resolveModule = (moduleName) => {
+    // Check if there's a local version of the module
+    try {
+      const path = `next/node_modules/${moduleName}`
```

Comment:
I meant this looks wrong because it runs on `next`, not on application. I think this is not what you intended.